Services/UI/OJS.Services.Ui.Business/Validation/Implementations/SubmissionDetailsValidationService.cs

```diff
@@ -0,0 +1,30 @@
+﻿namespace OJS.Services.Ui.Business.Validation.Implementations;
+
+using OJS.Services.Common.Models;
+using OJS.Services.Common.Models.Users;
+using OJS.Services.Ui.Models.Submissions;
+
+public class SubmissionDetailsValidationService : ISubmissionDetailsValidationService
+{
+    public ValidationResult GetValidationResult((SubmissionDetailsServiceModel?, UserInfoModel) item)
```

Comment:
argument - 'item' should be named correctly

Reply:
Renamed to validationInput.